src/popup/pages/AccountInfo/index.js

```diff
@@ -266,6 +246,7 @@ class AccountInfo extends React.Component {
           <>
             <div className="account-info-container">
               {this.renderCommonShowItem(getLanguage("accountAddress"), showAddress, this.copyAddress)}
+              {evmAddress && this.renderCommonShowItem(getLanguage("evmAddress"), evmAddress, this.copyAddress)}
```

Comment:
here it uses `copyAddress`, which copies the oasis address.

Reply:
i fix this in this [commit ](https://github.com/oasisprotocol/oasis-wallet-ext/pull/168/commits/879a33ac51f317c0af846fad4c379c06883f5a43)

Reply:
:+1: resolving